test/test_manifest.json

```diff
@@ -5907,5 +5907,14 @@
        "link": true,
        "forms": true,
        "type": "eq"
+    },
+    {  "id": "issue14021",
+       "file": "pdfs/issue14021.pdf",
+       "md5": "d18aa84135ce985c70a8f56306ecb95f",
+       "link": true,
+       "form": true,
```

Comment:
This should be `"forms": true` instead to work as intented; also can you please move this down to below the `type`-line instead since that helps to quickly see exactly what sort of test it is?